openedx/core/djangoapps/notifications/utils.py

```diff
@@ -31,24 +28,6 @@ def find_pref_in_normalized_prefs(pref_name, app_name, prefs_list):
     return None
 
 
-def get_show_notifications_tray(user):
-    """
-    Returns show_notifications_tray as boolean for the courses in which user is enrolled
-    """
-    show_notifications_tray = False
-    learner_enrollments_course_ids = CourseEnrollment.objects.filter(
-        user=user,
-        is_active=True
-    ).values_list('course_id', flat=True)
-
-    for course_id in learner_enrollments_course_ids:
-        if SHOW_NOTIFICATIONS_TRAY.is_enabled(course_id):
```

Comment:
This function checks if waffle_flag is enabled for any course. It cannot be replaced with waffle_flag.is_enabled. Use `ENABLE_NOTIFICATIONS` in place of `SHOW_NOTIFICATIONS_TRAY`.